src/js/modules/sections/components/SectionForm.js

```diff
@@ -5,9 +5,12 @@ import React, { Component } from 'react'
 import Paper from 'material-ui/Paper'
 import TextField from 'material-ui/TextField'
 import RaisedButton from 'material-ui/RaisedButton'
+import SelectField from 'material-ui/SelectField'
+import MenuItem from 'material-ui/MenuItem'
 import injectSheet from 'react-jss'
 import { createSection } from '../actions'
 import { connect } from 'react-redux'
+import { topLevelSectionsSelector } from '../selectors'
```

Comment:
Just a reminder about one of my first comments: double quotes vs. single quotes.